### PR TITLE
EOS-26484: Support http request to query byte count for CSM

### PR DESCRIPTION
**Pull request by rahul27kumar**
```
Hare fetches bytecount data from Motr. This needs to be persisted and
an interface must be provided to query the same.

Solution:
- Add Consul KVs for healthy,degraded,critical and damaged 
   as a part of cfgen.
- Add a hctl status sub command `--byecount` to report bytecount
   information.
- Provide a http command to query byte count information
   http://hax-svc:8008/v1/cluster/status/bytecount
```

Signed-off-by: Rahul Kumar <rahul.kumar@seagate.com>

### CODE REVIEW

**Comment by rahul27kumar** [comment]
docker custom build :
[build.log](https://github.com/Seagate/cortx-hare/files/7863254/build.log)

deployment log: 
[deployment.log](https://github.com/Seagate/cortx-hare/files/7863267/deployment.log)

[root@ssc-vm-g2-rhev4-2813 ~]# curl http://172.16.208.207:8008/v1/cluster/status/bytecount
{
  "byte_count": {
    "critical_byte_count": 0,
    "damaged_byte_count": 0,
    "degraded_byte_count": 0,
    "healthy_byte_count": 0
  }
}

**Comment by mssawant** [comment]
@rahul27kumar, please update the commit message as,
```
EOS-26484: support http request to query byte count for CSM 

Hare fetches bytecount data from Motr. This needs to be persisted and
an interface must be provided to query the same.

Solution:
- Add Consul KVs for healthy,degraded,critical and damaged 
   as a part of cfgen.
- Add a hctl status sub command `--byecount` to report bytecount
   information.
- Provide a http command to query byte count information
   http://hax-svc:8008/v1/cluster/status/bytecount

Signed-off-by: Rahul Kumar <rahul.kumar@seagate.com>
```

**Review by mssawant — APPROVED** [review]
Looks good. Kindly address the minor commit message comment.

File: utils/hare-status [hunk 1]
```diff
@@ -99,6 +100,15 @@ def leader_tag(cns: Consul, host: str) -> str:
     return ' (RC)' if kv_value_as_str(cns, 'leader') == host else ''
 
 
+def byte_count(cns: Consul) -> List[Byte_count]:
```
Comment (suvratjoshi) [inline utils/hare-status:103]: Can you please add documentation for all the newly added functions

**Review by vaibhavparatwar — APPROVED** [review]

**Comment by rahul27kumar** [comment]
After adding comments generated custom build and deployment testing:
[EOS-26484.log](https://github.com/Seagate/cortx-hare/files/7896059/EOS-26484.log)

**Review by suvratjoshi — APPROVED** [review]

**Pull request merged**